[2024-09-06T00:00:07.981+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-06T00:00:08.043+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-09-06T00:00:08.065+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-05T00:00:00+00:00 [queued]>
[2024-09-06T00:00:08.067+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-09-06T00:00:08.116+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): first_task> on 2024-09-05 00:00:00+00:00
[2024-09-06T00:00:08.149+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=27600) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-06T00:00:08.152+0000] {standard_task_runner.py:72} INFO - Started process 27605 to run task
[2024-09-06T00:00:08.144+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_operator_dag_v1', 'first_task', 'scheduled__2024-09-05T00:00:00+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpi1815ltf']
[2024-09-06T00:00:08.158+0000] {standard_task_runner.py:105} INFO - Job 162: Subtask first_task
[2024-09-06T00:00:08.265+0000] {task_command.py:467} INFO - Running <TaskInstance: python_operator_dag_v1.first_task scheduled__2024-09-05T00:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-06T00:00:08.457+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='python_operator_dag_v1' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-05T00:00:00+00:00'
[2024-09-06T00:00:08.461+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-06T00:00:08.521+0000] {logging_mixin.py:190} INFO - The sum is 101
[2024-09-06T00:00:08.525+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-06T00:00:08.563+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-06T00:00:08.565+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_operator_dag_v1, task_id=first_task, run_id=scheduled__2024-09-05T00:00:00+00:00, execution_date=20240905T000000, start_date=20240906T000008, end_date=20240906T000008
[2024-09-06T00:00:08.682+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-06T00:00:08.719+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-06T00:00:08.723+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
